src/applications/static-pages/static-pages-entry.js

```diff
@@ -52,6 +55,7 @@ Sentry.configureScope(scope => scope.setTag('source', 'static-pages'));
 const store = createCommonStore({
   ...facilityReducer,
   ...findVaFormsWidgetReducer,
+  ...post911GIBillStatusReducer,
```

Comment:
This is why I say we'll probably need a better pattern for reducers required by widgets.